### ISSUE TITLE
Doesn't work with button:card

### ISSUE THREAD

**Issue by IGOLz** [PiotrMachowski/Home-Assistant-Lovelace-Local-Conditional-card#30]
Hi, thanks for this awesome card but i have a problem making it work with button card.

The card works perfectly when i use it with almost anything else, but when i put a custom:button-card inside it doesn't show the card.

Here's an example:
![image](https://github.com/user-attachments/assets/bb671cc4-c409-442e-98ee-448a250cefea)

This is the code that's not working:
```
type: custom:local-conditional-card
id: tab1
default: show
card:
  type: custom:button-card
  entity: sensor.valvola_tado_bagno_camera_info_heating
```

Example of code working:
```
type: custom:local-conditional-card
id: tab1
default: show
card:
  type: alarm-panel
  entity: alarm_control_panel.alarmo
```

Can you fix it please?

**Comment by IGOLz** [comment]
For anyone having this problem, i fixed it by wrapping the button-card inside a grid like this:

```
type: custom:local-conditional-card
default: show
id: tab1
card:
  square: false
  type: grid
  cards:
    - type: custom:button-card
      entity: sensor.valvola_tado_bagno_camera_info_heating
```

And now it's working perfectly!

**Comment by PiotrMachowski** [comment]
Fixed in [v2.1.1](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-Local-Conditional-card/releases/tag/v2.1.1)
